src/run.ts

```diff
@@ -26,18 +20,18 @@ const run = async () => {
   logger.info(`Starting to gather dependency info`)
 
   const components = await getComponents(config.serviceCatalogueUrl)
-  const dependencies = await gatherComponentDependencies(config.environments.dev, components)
-  const componentMap = components.getComponentMap(dependencies)
+  const dependencies = await getDependencies(config.environments.dev)
+  const componentMap = components.buildComponentMap(dependencies)
 
-  const { categoryToComponent, componentDependencyInfo } = await gatherDependencyInfo(componentMap)
+  const { categoryToComponent, componentDependencyInfo, missingServices } = gatherDependencyInfo(componentMap)
+
+  logger.info(`Services missing from service catalogue: \n\t${missingServices.join('\n\t')}`)
```

Comment:
this is interesting info - lists missing services/environments that aren't in the service catalogue.

There's quite a few